src/functions/createAudit.ts

```diff
@@ -67,6 +68,8 @@ export function createAudit(
                     restaurantBugets[index] - restaurantBugets[0] - tips.flat().reduce((a, b) => a + b, 0)
                 }, recived in tips ${tips.flat().reduce((a, b) => a + b, 0)}`
             );
+            output.push(`Daily tax from orders to pay: ${dailyTax[0]}`);
+            output.push(`Daily tax from tips to pay: ${dailyTax[1]}`);
```

Comment:
Why do we use array for hardcoded indexes?
Is there possibility that array will have more or less than 2 elements?

Reply:
With current implementation it's not possible.

Reply:
Do you expect more elements in future? Or less?